### ISSUE TITLE
Flipping the block in the GUI should not flip the block's picture

### ISSUE THREAD

**Issue by zdebanos** [robertobucher/pysimCoder#75]
Make the block flip such that the svg won't flip with it.

**Comment by robertobucher** [comment]
You're right! I have to check the last commit: it increase the quality of the graphic but with this problem.
I'll check it ASAP

**Comment by zdebanos** [comment]
Thank you for your answer. I may look into it as well when I have time. I just put it here to not forget about it.

**Comment by robertobucher** [comment]
I've put a temporary solution to avoid the flipping of the images in blocks.
I think that instead of using the "/tmp" folder we can try to implement a ram disk in the future or setting the rendering of the block in the block class and modifying it only by some actions...

**Comment by zdebanos** [comment]
Thank you very much Roberto. I was thinking about doing the same thing, however using something more pythonic. I've come across svgutils python package, which could prove useful. However, it only works with files and I am trying to get a simple pull request into its mainline that could work with the xml loaded as a string so there would be no need to save it as a tmpfile (https://github.com/btel/svg_utils/pull/129). 

Svgutils package has a scale method which can flip the image. It is also much more simple and doesn't require inkscape to run.

**Comment by robertobucher** [comment]
I haven't found how to use svg_utils (no examples...), but this is of course a better solution as using "inkscape"! 

Let me know when you find how to use this package!

**Comment by robertobucher** [comment]
Please consider that we have to work usually with packages available in distros, and avoid when possible to have to many gits.
